capstone/src/main/java/com/google/univiz/api/SearchResource.java

```diff
@@ -11,5 +11,5 @@
    * Take a list of suggestion data and returns a list of Suggestions objects to the caller. This
    * object contains all data relevant to the Search feature.
```

Comment:
Now that you've fixed the parameter type, the javadoc also needs to be updated.

Reply:
updated!